Docker.DotNet/Properties/AssemblyInfo.cs

```diff
@@ -29,5 +29,5 @@
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.*")]
 
-[assembly: AssemblyVersion("1.0.0.0")]
-[assembly: AssemblyFileVersion("1.0.0.0")]
\ No newline at end of file
+[assembly: AssemblyVersion("1.16.0.0")]
+[assembly: AssemblyFileVersion("1.16.0.0")]
```

Comment:
I will try to get after it later...I'm currently waiting for my daughter to come out of surgery so it'll be a while before I can get to my laptop.

-----Original Message-----
From: "Ahmet Alp Balkan" <notifications@github.com>
Sent: ‎8/‎20/‎2015 3:21 PM
To: "ahmetalpbalkan/Docker.DotNet" <Docker.DotNet@noreply.github.com>
Cc: "Josh Garverick" <josh.garverick@gmail.com>
Subject: Re: [Docker.DotNet] Adding new NuSpec file with Microsoft.Bcl.Asyncdependency. (#19)

In Docker.DotNet/Properties/AssemblyInfo.cs:
> \ No newline at end of file
> +[assembly: AssemblyVersion("1.16.0.0")]
> +[assembly: AssemblyFileVersion("1.16.0.0")]
@jgarverick hmm not really actually. we're doing that using the nuget pkg version. If you could revert only this file I'd be happy to merge right away.
—
Reply to this email directly or view it on GitHub.